homeassistant/components/upnp/.translations/en.json

```diff
@@ -1,28 +1,13 @@
 {
     "config": {
         "abort": {
-            "already_configured": "UPnP/IGD is already configured",
```

Comment:
Why are parts of this file, and some others, changed? It seems unrelated to your other changes.

Reply:
@balloob I haven't had the time to look into the requested changes. I have looked into how they do things in openHAB to differentiate between NHC2 and NHC1. I've started on building this into my nhc2-coco lib. It does a UDP Broadcast on port 10000. Then controllers can respond during a 30sec period. It will generate a list with IP(s) and NHC version(s). We then make sure we have IPs too, and lookup the corresponding version? I could use some help on this...